### PR TITLE
Contact info scraper

### PR DESCRIPTION
**Pull request by JoeHarstad**

### CODE REVIEW

File: public/js/tsHttpInterceptor.js [hunk 1]
```diff
@@ -9,7 +9,11 @@
             var responseListeners = __responseIntercepts.filter(rl => rl.method === method.toLowerCase() && url.toLowerCase().indexOf(rl.urlContains) >= 0);
             responseListeners.forEach((rl) => {
                 if (rl.callBack) {
-                    rl.callBack({response: this.response, responseText: this.responseText, url: this.responseURL});
+                    let text = "";
```
Comment (emomon23) [inline public/js/tsHttpInterceptor.js:12]: const text = this.responseType === 'text' || this.responseType === ''? this.responseText : "";

File: public/linkedin/js/scrapers/linkedInContactInfoScraper.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+(() => {
+    const _scrapeContactInfo = async (cachedCandidate) => {
+        if (!cachedCandidate.isContactInfoScraped) {
+            cachedCandidate.isContactInfoScraped = true;
+            
+            const contactInfoLink = $(linkedInSelectors.publicProfilePage.CONTACT_INFO_LINK);
+            const contactInfoWindow = window.open($(contactInfoLink).attr("href"));
+            await tsCommon.sleep(5000);
```
Comment (emomon23) [inline public/linkedin/js/scrapers/linkedInContactInfoScraper.js:8]: use randomSleep?

File: public/linkedin/js/scrapers/linkedInPublicProfileScraper.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+            return updatedCandidate;
+        } else {
+            tsCommon.log("Public Profile Scraper:  Could not locate cached candidate to update.", "WARN");
+        }
+        
+        return null;
+    }
+
+    class LinkedInPublicProfileScraper {
+        scrapeProfile = _scrapeProfile;
```
Comment (emomon23) [inline public/linkedin/js/scrapers/linkedInPublicProfileScraper.js:54]: Scrape all their job experience too!

cachedCandidate should already have a .positions array, fill in each position with their position description text.

Also.  What do you think about just merging all of the positions descriptions into one big string and saving that, then we can just to a search on that big string?

eg.  candidate.rawExperienceDescriptionString = _joinAllPositionDescriptions()?

**Review by emomon23 — CHANGES_REQUESTED** [review]

File: public/linkedin/js/scrapers/linkedInPublicProfileScraper.js (hunk 3 above)
Comment (JoeHarstad) [reply public/linkedin/js/scrapers/linkedInPublicProfileScraper.js:54]: Add this level of scraping to the board of to-dos.  Scope here was contact info, but this is a good idea and I did consider it, just didn't want to balloon the scope when we have a grading initiative at hand as well, just wanted to clean up work-in-progress before it's all stale.

**Review by emomon23 — APPROVED** [review]

**Pull request merged**